File: wertyu.py
kb = ["`1234567890-=",
      "QWERTYUIOP[]\\",
      "ASDFGHJKL;'",
      "ZXCVBNM,./"]

map = {" ": " "}

for row in kb:
    lr = list(row)
    for i in range(1, len(lr)):
        map[lr[i]] = lr[i-1]

while True:
    try:
        line = input()
    except EOFError:
        break

    if len(line) == 0: break
    
    ll = list(line)
    for i in range(len(ll)):
        print(map[ll[i]], end="")
    print()

File: wertyu.cpp
#include <iostream>
#include <vector>
#include <unordered_map>

using namespace std;

int main()
{
    vector<string> kb = {
        "`1234567890-=",
        "QWERTYUIOP[]\\",
        "ASDFGHJKL;'",
        "ZXCVBNM,./"
    };

    unordered_map<char, char> conv;
    conv[' '] = ' ';

    for(auto row: kb)
    {
        for(int i = 1 ; i < row.size() ; ++i)
        {
            conv[row[i]] = row[i-1]; 
        }
    }

    string line;
    while(getline(cin, line) && !line.empty()) {
        for(auto c: line)
        {
            cout << conv[c];
        }
        cout << '\n';
    }
}
